hooks/useKeyboard/useKeyboard.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// eslint-disable-next-line simple-import-sort/imports
```

Comment:
it is better to use utility function if you need callback to be triggered. You should create a util addKeyboardListrener('Enter', () => { /* your code here */})